rmw_fastrtps_cpp/src/rmw_client.cpp

```diff
@@ -143,12 +145,12 @@ rmw_create_client(const rmw_node_t * node,
   publisherParam.topic.topicName += "Request";
 
 #ifdef DEBUG_LOGGING
-  fprintf(stderr, "************ Client Details *********\n");
-  fprintf(stderr, "Sub Topic %s\n", subscriberParam.topic.topicName.c_str());
-  fprintf(stderr, "Sub Partition %s\n", subscriberParam.qos.m_partition.getNames()[0].c_str());
-  fprintf(stderr, "Pub Topic %s\n", publisherParam.topic.topicName.c_str());
-  fprintf(stderr, "Pub Partition %s\n", publisherParam.qos.m_partition.getNames()[0].c_str());
-  fprintf(stderr, "***********\n");
+  RCUTILS_LOG_DEBUG("************ Client Details *********")
+  RCUTILS_LOG_DEBUG("Sub Topic %s", subscriberParam.topic.topicName.c_str())
+  RCUTILS_LOG_DEBUG("Sub Partition %s", subscriberParam.qos.m_partition.getNames()[0].c_str())
+  RCUTILS_LOG_DEBUG("Pub Topic %s", publisherParam.topic.topicName.c_str())
+  RCUTILS_LOG_DEBUG("Pub Partition %s", publisherParam.qos.m_partition.getNames()[0].c_str())
+  RCUTILS_LOG_DEBUG("***********")
```

Comment:
oh I thought that the severity levels were not observed. Thanks I'll remove them then!